app/Resources/doc/changeLog/CHANGELOG-0.1.4.md

```diff
@@ -13,7 +13,8 @@ Url to see changes :
  - [Theme bundle](https://github.com/open-orchestra/open-orchestra-theme-bundle/compare/v0.1.3...v0.1.4)
 
 ## Possible BC breaker
-
+- The ContactController in the DisplayBundle has been removed. Therefore there are no more controller in
```

Comment:
Well, i tought so, but :
* it's clearly a BC breaker
* it's not a configuration key that changed, it just may impact your configuration